src/Authentication/Authentication.Core/Microsoft.Graph.Authentication.Core.csproj

```diff
@@ -12,17 +12,18 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.24.0" />
     <PackageReference Include="Azure.Identity" Version="1.6.0" />
-    <PackageReference Include="Microsoft.Graph.Core" Version="2.0.8" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.43.1" />
-    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.20.0" />
-    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="6.17.0" />
-    <PackageReference Include="Microsoft.IdentityModel.Logging" Version="6.17.0" />
-    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.17.0" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.17.0" />
+    <PackageReference Include="Azure.Identity.BrokeredAuthentication" Version="1.0.0-beta.2" />
```

Comment:
You'll also need to add `Azure.Identity.BrokeredAuthentication` to `Microsoft.Graph.Authentication.nuspec` for it to be packaged with the module as nuget package:

https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/c1e324e36c535ccbe993ca9d46a70770375df937/src/Authentication/Authentication/Microsoft.Graph.Authentication.nuspec#L28-L31

Once done, run ` .\tools\GenerateAuthenticationModule.ps1 -Build -Pack -Run` to test the change.